src/react-components/ory/user-error-card.tsx

```diff
@@ -44,16 +45,21 @@ export const UserErrorCard = ({
         </h2>
       }
       image={cardImage}
+      size="wide"
     >
       <div
         className={gridStyle({ gap: 32, direction: "column" })}
-        data-testid={`ui/error/message`}
+        data-testid="ui/error/message"
       >
-        <Message severity="error">
-          An error occurred with the following message:&nbsp;
-          {status < 500 && message}
-        </Message>
-        {status >= 500 && <CodeBox data-testid={"code-box"}>{message}</CodeBox>}
+        {!is500 && (
+          <Message severity="error">
+            An error occurred with the following message:&nbsp;
+            {err.reason}
+          </Message>
+        )}
+        <CodeBox data-testid="code-box" toggleText="Error details">
+          {JSON.stringify(error, null, 2)}
+        </CodeBox>
```

Comment:
We now always include the error details, even for 400 errors, because they might be useful during development. For production apps, this might be a bit much. We could control that using a CSS variable?

Any other ideas?

Reply:
I think show all is fine for now, but i think there are some kratos e2e tests that would need to be adjusted (maybe) because of this change.

Reply:
I'll check once this is merged, and I bump it in the examples repo.